MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include README.rst
 include CHANGES.rst
 include LICENSE.rst
 recursive-include licenses *
+include astropy_helpers\commands\src\compiler.c
```

Comment:
I need to investigate why https://github.com/astropy/astropy-helpers/blob/master/setup.cfg isn't working